inspect.lua

```diff
@@ -78,7 +78,11 @@ replacer.inspect = function(_, user, pointed_thing, mode, show_receipe)
 			end
 
 		end
-		text = text..' at '..minetest.pos_to_string(ref:getpos())
+		if ref:getpos() then
+			text = text..' at '..minetest.pos_to_string(ref:getpos())
```

Comment:
The `getpos()` function is deprecated (it was there before this PR but would be a good improvement anyway).
```suggestion
		if ref:get_pos() then
			text = text..' at '..minetest.pos_to_string(ref:get_pos())
```